packages/dart/noports_core/lib/src/sshnp/util/sshnpd_channel/sshnpd_channel.dart

```diff
@@ -255,39 +255,86 @@ abstract class SshnpdChannel with AsyncInitialization, AtClientBindings {
   /// and corresponding info, and a list of active devices (devices which also
   /// responded to our real-time ping).
   Future<SshnpDeviceList> listDevices() async {
```

Comment:
This function wraps the previous logic, deferring to the old logic if -t is set, or aggregating across all devices if -t is not set.